### DISCUSSION TITLE
compression_quality applies to photo

### DISCUSSION THREAD

**Discussion by QVaKor** [LycheeOrg/Lychee#2357] [Bugs] [answered]
### Lychee version

v5.2.0

### Which PHP version are you using?

PHP 8.2

### Detailed description of the problem

When uploading photo with compression_quality set to <100 main image is compressed as well not only thumbnail like setting description states

### Steps to reproduce the issue

1. Set compression_quality to < 100
2. Upload photo (I tested that on ~13 Mb file)
3. Size and quality of photo is decreased (with compression_quality set to 90 photo after upload was 3.3 Mb)

### Diagnostics [REQUIRED]

  Diagnostics
    -----------
        Warning: Dropbox import not working. dropbox_key is empty.
        Info: Latest version of PHP is 8.3
        Warning: git (software) is not available.
        Info: Found 1 medium that could be generated.
              You can use `php artisan lychee:generate_thumbs medium 1` to generate them.
    
    System Information
    ------------------
        Lychee Version (tag):                    v5.2.0 (31a394b) -- Up to date (28 minutes ago).
        DB Version:                              5.2.0
        
        composer install:                        --no-dev
        APP_ENV:                                 production
        APP_DEBUG:                               false
        APP_URL:                                 set
        APP_DIR:                                 default
        LOG_VIEWER_ENABLED:                      true
        LIVEWIRE_ENABLED:                        true
        
        System:                                  Linux
        PHP Version:                             8.2.7
        PHP User agent:                          Lychee/4 (https://lycheeorg.github.io/)
        Timezone:                                Europe/Warsaw
        Max uploaded file size:                  100M
        Max post size:                           100M
        Livewire chunk size:                     12.00 MB
        Max execution time:                      3600
        MySQL Version:                           10.11.7-MariaDB-1:10.11.7+maria~ubu2204
        
        exec() Available:                        yes
        Imagick Available:                       1
        Imagick Enabled:                         1
        Imagick Version:                         1691
        GD Version:                              2.3.3
        Number of foreign key:                   12 found.
    
    Config Information
    ------------------
        version:                                 050200
        check_for_updates:                       0
        sorting_photos_col:                      taken_at
        sorting_photos_order:                    ASC
        sorting_albums_col:                      max_taken_at
        sorting_albums_order:                    ASC
        imagick:                                 1
        skip_duplicates:                         1
        small_max_width:                         0
        small_max_height:                        360
        medium_max_width:                        1920
        medium_max_height:                       1080
        lang:                                    en
        image_overlay_type:                      exif
        default_license:                         none
        compression_quality:                     100
        grants_full_photo_access:                1
        delete_imported:                         0
        mod_frame_enabled:                       1
        mod_frame_refresh:                       30
        thumb_2x:                                1
        small_2x:                                1
        medium_2x:                               1
        landing_page_enable:                     0
        site_owner:                              Gabriel Szeloch
        landing_title:                           John Smith
        landing_subtitle:                        Cats, Dogs & Humans Photography
        sm_facebook_url:                         https://www.facebook.com/JohnSmith
        sm_flickr_url:                           https://www.flickr.com/JohnSmith
        sm_twitter_url:                          https://www.twitter.com/JohnSmith
        sm_instagram_url:                        https://instagram.com/JohnSmith
        sm_youtube_url:                          https://www.youtube.com/JohnSmith
        landing_background:                      dist/cat.webp
        site_title:                              Private Photo Gallery
        footer_show_copyright:                   1
        site_copyright_begin:                    2023
        site_copyright_end:                      2023
        footer_additional_text:                  
        footer_show_social_media:                0
        search_public:                           1
        SL_enable:                               0
        SL_for_admin:                            0
        recent_age:                              1
        grants_download:                         0
        photos_wraparound:                       1
        map_display:                             0
        zip64:                                   1
        map_display_public:                      0
        map_provider:                            Wikimedia
        force_32bit_ids:                         0
        map_include_subalbums:                   0
        update_check_every_days:                 3
        has_exiftool:                            1
        share_button_visible:                    1
        import_via_symlink:                      0
        has_ffmpeg:                              1
        location_decoding:                       0
        location_decoding_timeout:               30
        location_show:                           1
        location_show_public:                    0
        rss_enable:                              0
        rss_recent_days:                         7
        rss_max_items:                           100
        prefer_available_xmp_metadata:           0
        editor_enabled:                          1
        lossless_optimization:                   0
        swipe_tolerance_x:                       150
        swipe_tolerance_y:                       250
        local_takestamp_video_formats:           .avi|.mov
        log_max_num_line:                        1000
        unlock_password_photos_with_url_param:   0
        nsfw_visible:                            1
        nsfw_blur:                               0
        nsfw_warning:                            0
        nsfw_warning_admin:                      0
        nsfw_banner_override:                    
        map_display_direction:                   1
        album_subtitle_type:                     oldstyle
        upload_processing_limit:                 4
        new_photos_notification:                 0
        legacy_id_redirection:                   1
        zip_deflate_level:                       -1
        SA_enabled:                              0
        default_album_protection:                1
        album_decoration:                        layers
        album_decoration_orientation:            row
        allow_username_change:                   1
        auto_fix_orientation:                    1
        use_job_queues:                          0
        random_album_id:                         starred
        use_last_modified_date_when_no_exif_date: 0
        ffmpeg_path:                             /usr/bin/ffmpeg
        ffprobe_path:                            /usr/bin/ffprobe
        layout:                                  justified
        date_format_photo_thumb:                 M j, Y, g:i:s A e
        date_format_photo_overlay:               M j, Y, g:i:s A e
        date_format_sidebar_uploaded:            M j, Y, g:i:s A e
        date_format_sidebar_taken_at:            M j, Y, g:i:s A e
        date_format_hero_min_max:                F Y
        date_format_hero_created_at:             M j, Y, g:i:s A T
        date_format_album_thumb:                 M Y
        upload_chunk_size:                       0
        nsfw_banner_blur_backdrop:               0
        search_pagination_limit:                 1000
        search_minimum_length_required:          4
        photo_layout_justified_row_height:       320
        photo_layout_masonry_column_width:       300
        photo_layout_grid_column_width:          250
        photo_layout_square_column_width:        200
        photo_layout_gap:                        12
        display_thumb_album_overlay:             always
        display_thumb_photo_overlay:             hover
        default_album_thumb_aspect_ratio:        1/1
        use_album_compact_header:                0
        login_button_position:                   left
        back_button_enabled:                     0
        back_button_text:                        Return to Home
        back_button_url:                         /
        current_job_processing_visible:          1

### Browser & System [REQUIRED]

Microsoft Edge version 123.0.2420.81
Windows 10

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by ildyria** [comment]
Thank you I will double check that.

**Comment by ildyria** [comment, marked as answer]
Fixed on V6.
